File: 2021/src/test/java/nel/marco/Day1Test.java
package nel.marco;

import org.junit.jupiter.api.Test;

import java.io.IOException;

import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertNotEquals;

public class Day1Test {


    @Test
    public void partOne() throws IOException {
        String answer = Day1.partOne();

        assertNotEquals("-1", answer);
        assertEquals("1316", answer);

    }

    @Test
    public void partTwo() throws IOException {

        String answer = Day1.partTwo();

        assertNotEquals("-1", answer);
        assertEquals("1344", answer);
    }
}

File: 2019/src/test/kotlin/nel/marco/Day1Test.kt
package nel.marco

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test
import java.nio.file.Files
import java.nio.file.Path

class Day1Test {

    val input = Files.readAllLines(Path.of("C:\\code\\AdventOfCode\\2019\\src\\main\\resources\\Day1"))
    val day = Day1()

    @Test
    fun part1() {
        assertEquals("3474920", day.part1(input))
    }

    @Test
    fun part2() {
        assertEquals("5209504", day.part2(input))
    }
}